misc/config_tools/config_app/static/main.js

```diff
@@ -790,7 +789,7 @@ function create_setting(type, default_name, name, mode){
             create_flag = true
             if(exist == "yes") {
                 overwirte_confirm_message = 'Setting name: ' + create_config['create_name'] + ' existed in ' +
```

Comment:
the word overwrite_ is misspelled as overwirte_ in the code here

Reply:
Thanks for the reminder. updated this commit and fixed the typo.